sale_service_project/models/product.py

```diff
@@ -34,13 +34,15 @@ def action_compute_price(self):
 
 class ProductTaskWork(models.Model):
     _name = 'product.task.work'
+    _order = 'sequence'
 
     product_id = fields.Many2one(
         comodel_name='product.template', string='Product', ondelete='restrict')
     name = fields.Char(string='Name')
     hours = fields.Float(
         string='Hours',
-        digits_compute=dp.get_precision('Product Unit of Measure'))
+        digits=dp.get_precision('Product UoS'))
```

Comment:
The name of the digits precision is 'Product Unit of Measure', so it was correct before (although the function is `digits`, not `digits_compute`)